Index: profiles/openscholar/modules/contrib/og_vocab/includes/og_vocab.og_vocab.inc
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- includes/og_vocab.og_vocab.inc	(revision )
+++ includes/og_vocab.og_vocab.inc	(revision )
@@ -329,7 +329,10 @@
         $ids[] = $value;
       }
     }
-    form_set_value($form[$field_name][LANGUAGE_NONE], $ids, $form_state);
+    $parents = $form_state['field'][$field_name][LANGUAGE_NONE]['array_parents'];
+    array_pop($parents);
+    drupal_array_set_nested_value($form_state['values'], $parents, $ids, TRUE);
+    //form_set_value($form[$field_name][LANGUAGE_NONE], $ids, $form_state);
   }
 }
 
